File: felipe/Problema0190/problema0190.js
const prompt = require("prompt-sync")({ sigint: true })

lim = prompt("Digite o limite superior maior que 0: ")
incr = prompt("Digite o incremento, maior que 0: ")

for (let i = 0; i <= lim/incr; i++) {
    console.log(i*incr+" ");
}

File: felipe/Problema0190/problema0190.java
import java.util.Scanner;

public class problema0190{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int lim, incr;
        
        System.out.print("Digite o limite superior maior que 0: ");
        lim = Integer.parseInt(input.nextLine());

        System.out.print("Digite o incremento, maior que 0: ");
        incr = Integer.parseInt(input.nextLine());

        for(int i = 0; i <= lim/incr; i++){
            System.out.print(i*incr+" ");
        }
    }
}